templates/jobs/confirm_submission.html

```diff
@@ -6,11 +6,14 @@
         <div class="row">
             <div class="col-md-12">
                 {% include 'includes/_info_messages.html' %}
-                <h2>Thank you for your submission!</h2>
-                <p><strong>Please consider making a donation for our cause!</strong> Just take a look at Django Girls <a href="https://www.patreon.com/djangogirls">patreon site.</a></p>
-
-                <p>"Django Girls is a non-profit organization that empowers and helps women to organize free, one-day programming workshops by providing tools, resources and support.(...)
-                All of the money gathered here on Patreon will go towards establishing a Django Girls non-profit organization as well as supporting the future growth of Django Girls, allowing us to create more learning resources and enabling even more women to discover the joy of programming."</p>
+                <h3>Yay, high-fives! Thank you for adding a job offer/meetup.</h3>
+                <p> 
+                    We carefully review each offer and meetup. We will get in touch with you shortly. 
+                    Would you help us out, too?
+                </p>
+                <p>
+                    We don’t charge for helping you reach candidates for your job offer. However, we are a non-profit organisation with a mission to show more women the joy of programming. Consider making a <a href="https://www.patreon.com/djangogirls">small donation</a> to help us make more impact on the world. Thank you!
```

Comment:
"...helping you reach..."
"Please consider..."

We mention jobs and meetups on line 11 but only jobs here.  Is this template only for confirmation of job offers?